pkgs/development/libraries/openldap/default.nix

```diff
@@ -46,6 +46,7 @@ stdenv.mkDerivation rec {
   postBuild = ''
     make $makeFlags CC=$CC -C contrib/slapd-modules/passwd/sha2
     make $makeFlags CC=$CC -C contrib/slapd-modules/passwd/pbkdf2
+    make $makeFlags CC=$CC -C contrib/slapd-modules/passwd/argon2
```

Comment:
Does this build something multithreaded? Does it required -j$NIX_BUILD_CORES.

Reply:
I don't understand the question. This builds the argon2 slapd module the same way it builds the sha2 and pbkdf2 ones. Cleanups of the derivation in general should be a separate PR.

Reply:
> This builds the argon2 slapd module the same way it builds the sha2 and pbkdf2 ones.

So they are also missing this?

> Cleanups of the derivation in general should be a separate PR.

Its not like this is a pressing issue and we didn't have time for that. nixpkgs is so big that while we are rebuilding something and looking into a package we should also cleanup obvious things.

Reply:
I don't know. Isn't `-j$NIX_BUILD_CORES` part of `$makeFlags` already?

This PR added support for another slapd module, the same way other slapd modules were added here. The build system looks a bit arcane, and I can understand just another component was added the same way others are already in there.

If you want to do further cleanups, feel free to send a PR.